workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowInstancePageContent.tsx

```diff
@@ -140,6 +142,28 @@ export const WorkflowInstancePageContent: React.FC<{
     setIsVariablesDialogOpen(prev => !prev);
   }, []);
 
+  const generic = usePermission({
+    permission: orchestratorAdminView,
+  });
+
+  const viewVariables = !generic.loading && generic.allowed && (
```

Comment:
```suggestion
  const viewVariables = generic.allowed && (
```

if generic.allowed is true, then the loading is done as well.